client/src/hooks/useAddPlayerToTeam.js

```diff
@@ -1,8 +1,12 @@
 import { useMutation } from 'react-query'
+import useUserDetails from './useUserDetails'
+import { getTeamFromUser } from 'utils'
 import { addPlayerToTeam } from 'api'
 
 const useAddPlayerToTeam = (options = {}) => {
-    return useMutation(addPlayerToTeam, options)
+    const { user } = useUserDetails()
```

Comment:
This was in a recent PR but can you change this to get the teamId from the route instead of using the one from userDetails? It's more reliable. You can look in useLeague as a reference